Formula/nvidia-docker.rb

```diff
@@ -0,0 +1,53 @@
+require "language/go"
+
+class NvidiaDocker < Formula
+  desc "Build and run Docker containers leveraging NVIDIA GPUs"
+  homepage "https://github.com/NVIDIA/nvidia-docker"
+  url "https://github.com/NVIDIA/nvidia-docker/archive/v1.0.1.tar.gz"
+  sha256 "bb7cf057fadc5935c3ea2f7320516cae8feb8fc2421af9772e3127b258e38acd"
+
+  depends_on "go" => :build
+
+  patch :p1, :DATA
+
+  go_resource "golang.org/x/crypto" do
+    url "https://github.com/golang/crypto.git",
+        :revision => "adbae1b6b6fb4b02448a0fc0dbbc9ba2b95b294d"
+  end
+
+  def install
+    ENV["GOPATH"] = buildpath/"go"
+
+    (buildpath/"go/src/github.com/nvidia/nvidia-docker").install buildpath.children
+    Language::Go.stage_deps resources, buildpath/"go/src"
+
+    cd("go/src/github.com/nvidia/nvidia-docker/src/nvidia-docker") do
+      system "go", "build", "-o", bin/"nvidia-docker"
+    end
+  end
+
+  test do
+    system "#{bin}/nvidia-docker", "version"
```

Comment:
Can’t we write a test that does something more substantial than e.g. `help`  or `version`?